go.mod

```diff
@@ -5,38 +5,51 @@ require (
 	github.com/DataDog/zstd v1.3.5 // indirect
 	github.com/asdine/storm v0.0.0-20190216191021-fe89819f6282
 	github.com/decred/dcrd/addrmgr v1.0.2
-	github.com/decred/dcrd/blockchain/stake v1.1.0
-	github.com/decred/dcrd/chaincfg v1.4.0
-	github.com/decred/dcrd/chaincfg/chainhash v1.0.1
+	github.com/decred/dcrd/blockchain/stake/v2 v2.0.1
+	github.com/decred/dcrd/chaincfg/chainhash v1.0.2
+	github.com/decred/dcrd/chaincfg/v2 v2.2.0
 	github.com/decred/dcrd/connmgr v1.0.2
-	github.com/decred/dcrd/dcrec v0.0.0-20190402182842-879eebce3333
-	github.com/decred/dcrd/dcrutil v1.2.0
+	github.com/decred/dcrd/dcrec v1.0.0
+	github.com/decred/dcrd/dcrjson v1.2.0 // indirect
+	github.com/decred/dcrd/dcrutil v1.4.0
+	github.com/decred/dcrd/dcrutil/v2 v2.0.0
```

Comment:
Should probably look for all occurrences of dcrutil v1.4.0 and replace with dcrutil v2. `go mod tidy` afterwards.